autoload/slime.vim

```diff
@@ -2,23 +2,54 @@
 " Target Interface
 """""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""
 
+" TargetSend tries to send the text and its config to the right place, by
+" finding the first existing variable in this list and performing the
+" corresponding action.
+" 1. b:slime_target_send  -> Trying to execute a `system` call, passing the
+" configuration in a `CONFIG` environment variable,
+" 2. b:SlimeTargetSend    -> Calling the function with `config` and `text`,
+" 3. g:slime_target_send  ->  Trying to execute a `system` call, passing the
+" configuration in a `CONFIG` environment variable,
+" 4. g:SlimeTargetSend    -> Calling the function with `config` and `text`.
 function! s:TargetSend(config, text)
-  let output = system("CONFIG=" . shellescape(a:config) . " " . g:slime_target_send, a:text)
+  if exists("b:slime_target_send")
+    let output = system("CONFIG=" . shellescape(a:config) . " " . b:slime_target_send, a:text)
+  elseif exists("*b:SlimeTargetSend")||exists("b:SlimeTargetSend") 
```

Comment:
I guess we can agree that this is ugly, but I don't know how to do that cleaner and still allow for all proposed configurations. This should be safe though, since `SlimeTargetSend` cannot be anything else that a function.

Reply:
what's the difference between `*b:SlimeTargetSend` and `b:SlimeTargetSend`? (vimscript-wise)

Reply:
I took that from `:help exists`, it says the star at the beginning is for functions, and when there's nothing it is for internal variables.